ioctl/newcmd/did/didderegister.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package did
+
+import (
+	"math/big"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/newcmd/action"
+)
+
+// Multi-language support
+var (
+	_deregisterCmdUses = map[config.Language]string{
+		config.English: "deregister (CONTRACT_ADDRESS|ALIAS)",
+		config.Chinese: "deregister (合约地址|别名)",
+	}
+	_deregisterCmdShorts = map[config.Language]string{
+		config.English: "Deregister DID on IoTeX blockchain",
+		config.Chinese: "在IoTeX链上注销DID",
```

Comment:
add flag shorts.  also in did register 
https://github.com/iotexproject/iotex-core/blob/ee04a051811a9c23b33a291fb66c2de957b78bf3/ioctl/newcmd/action/actiontransfer.go#L29